### PR TITLE
doc(readme): update 7b/20b chat model information

### PR DESCRIPTION
**Pull request by ly015**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Update 7b/20b chat models.

## Modification

- Update news and model information in readme files
- Incorporate modifications by pre-commit hooks that are not directly relevant to the readme update.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

**Review by gaoyang07 — APPROVED** [review]

File: README-zh-Hans.md [hunk 1]
```diff
@@ -44,8 +44,8 @@ InternLM 是一个开源的轻量级训练框架，旨在支持大模型训练
 
 ## 更新
 
-[20230920] InternLM-20B 已发布，包括基础版和对话版。  
-[20230822] InternLM-7B-Chat v1.1 已发布，增加了代码解释器和函数调用能力。您可以使用 [Lagent](https://github.com/InternLM/lagent) 进行尝试。
```
Comment (gaoyang07) [inline README-zh-Hans.md:48]: 20230822的info是否需要删掉？

File: README-ja-JP.md [hunk 2]
```diff
@@ -43,26 +43,25 @@ InternLM は、70 億のパラメータを持つベースモデルと、実用
 
 ## 新闻
 
-InternLM-7B-Chat v1.1 は、コード インタプリタと関数呼び出し機能を備えてリリースされました。 [Lagent](https://github.com/InternLM/lagent) で試すことができます。
```
Comment (gaoyang07) [inline README-ja-JP.md:46]: 补充下日语部分？

File: README.md [hunk 3]
```diff
@@ -44,25 +44,23 @@ Based on the InternLM training framework, we have released two open-sourced pret
 
 ## News
 
-[20230920] InternLM-20B is released with base and chat versions.  
-[20230822] InternLM-7B-Chat v1.1 is released with code interpreter and function calling capability. You can try it with [Lagent](https://github.com/InternLM/lagent).
+[20231213] InternLM-7B-Chat and InternLM-20B-Chat checkpoints are updated. With an improved finetuning strategy, the new chat models can generate higher quality responces with greater stylistic diversity.
```
Comment (gaoyang07) [inline README.md:47]: fix typo: `responces` -> `responses`

File: README-zh-Hans.md (hunk 1 above)
Comment (ly015) [reply README-zh-Hans.md:48]: 因为后续不再区分7b模型的各种版本，1.1版模型在huggingface/modelscope上已经隐藏。这里还需要保留时间线么？

Comment (gaoyang07) [reply README-zh-Hans.md:48]: 那可以考虑去掉。

**Review by gaoyang07 — APPROVED** [review]

**Pull request merged**